### PR TITLE
Reconfigure package with zopefoundation/meta/config

### PR DESCRIPTION
**Pull request by dataflake**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,6 +1,13 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
+include *.rst
```
Comment (jugmac00) [inline MANIFEST.in:3]: Do we need this directive when later on we  do a recursive include?

Also not sure whether this should be raised here or in the meta repo.

**Review by jugmac00 — APPROVED** [review]
LGTM!

File: MANIFEST.in (hunk 1 above)
Comment (dataflake) [reply MANIFEST.in:3]: We need this because this includes `*.rst` at the root. The later recursive include applies to the `src` subfolder.

Comment (jugmac00) [reply MANIFEST.in:3]: Thank you!

**Pull request merged**